stats/views.py

```diff
@@ -1,5 +1,29 @@
 from django.http import HttpResponse
+from stats.losses.core import StatsService
+import json
 
+statsService = StatsService()
 
 def index(request):
-    return HttpResponse("Hello, world. You're at the polls index.")
\ No newline at end of file
+    print(request.body)
+
+    #set_up_code()
+
+    return HttpResponse("Hello, world. You're at the polls index.")
+
+
+def calculate(request):
+    statsService
+    body = json.loads(request.body.decode('utf-8'))
```

Comment:
this can throw exceptions with bad data

Reply:
Fixed!

Reply:
Will need to be more conscious of this going forward